zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -133,7 +133,13 @@ object ServerSpec extends HttpRunnableSpec(8088) {
         val res = app.requestBodyAsString(content = string)
         assertM(res)(equalTo(string.length.toString))
       }
-    }
+    } +
+      testM("POST Echo request body back") {
+        val path    = !! / "echoRequestBody"
+        val bodyMsg = "Hello World!"
+        val res     = request(path, Method.POST, bodyMsg).flatMap(_.getBodyAsString)
+        assertM(res)(equalTo(bodyMsg))
+      }
```

Comment:
why is this test added?

Reply:
In general, when we see our test cases, there are very few tests added for Methods like POST/PUT/DELETE also there are fewer examples with non GET methods.

Reply:
Try to only have 3 lines in any test.

Reply:
I wrote this specific test to simulate and fail the request.getBody

Reply:
Check the test written for `echo`

Reply:
Test for StaticApp are defined under `def staticAppSpec`.

Reply:
Done